docs/ci-integration/guides/gitlab-integration.md

```diff
@@ -24,6 +25,6 @@ earthly:
     - earthly --ci --push -P +build
 ```
 
-A full example is available [on GitLab](https://gitlab.com/brandon176/earthly-demo/-/tree/main).
+A full example is available [on GitLab](https://gitlab.com/earthly-technologies/earthly-demo).

```

Comment:
Was gitlab.com/earthly taken?

Reply:
Unfortunately yes. Although there's nothing on the page gitlab.com/earthly, when I try to register it, it says it's already taken 😞